test/pkg/environment/common/environment.go

```diff
@@ -157,10 +158,11 @@ func (env *Environment) DefaultNodeClass() *v1alpha1.KWOKNodeClass {
 
 func (env *Environment) DefaultNodePool(nodeClass *v1alpha1.KWOKNodeClass) *v1beta1.NodePool {
 	nodePool := test.NodePool()
+	gvk := object.GVK(nodeClass)
 	nodePool.Spec.Template.Spec.NodeClassRef = &v1beta1.NodeClassReference{
 		Name:       "default",
-		Kind:       "KWOKNodeClass",
-		APIVersion: v1alpha1.SchemeGroupVersion.Version,
+		Kind:       gvk.Kind,
+		APIVersion: gvk.Version,
```

Comment:
This is just the version and doesn't contain the group. It should have both

Reply:
Right, updated.